File: 99_Sol/BeakJoon/Bronze/Bronze 3/별찍기9.py
import sys
sys.stdin = open('input.txt')

N = int(input())

for j in range(N, 1, -1):
    print((N-j) * ' ' + (j * 2 - 1) * '*')

for i in range(1, N+1):
    print((N-i) * ' ' + (i * 2 - 1) * '*')

File: 99_Sol/BeakJoon/Bronze/Bronze 3/별찍기9.js
// 파일로부터 입력 받기
const input = require('fs').readFileSync('input.txt').toString().trim();

for (let N = input; N > 1; N--) {
    console.log(' '.repeat(input-N) + '*'.repeat(N * 2 - 1))
}

for (let N = 1; N <= input; N++) {
    console.log(' '.repeat(input-N) + '*'.repeat(N * 2 - 1))
}